src/Microsoft.ML.Data/Scorers/FeatureContributionCalculationTransform.cs

```diff
@@ -493,4 +599,79 @@ public void GetMetadata<TValue>(string kind, int col, ref TValue value)
             }
         }
     }
```

Comment:
The last step in adding FCC to the EntryPoints catalog is to add a public static method at the end here that takes the FCC arguments and returns one of the "CommonOutputs" as shown in [LogisticRegression](https://github.com/dotnet/machinelearning/blob/9db16c85888e7163c671543faee6ba1f47015d68/src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LogisticRegression.cs#L407) or in [OneHotVectorizer transformer](https://github.com/dotnet/machinelearning/blob/9db16c85888e7163c671543faee6ba1f47015d68/src/Microsoft.ML.Transforms/OneHotEncoding.cs#L283).  You basically want to call the constructor with the arguments and pass it to the CommonOutputs class.

Reply:
Looks good

---
In reply to: [241533831](https://github.com/dotnet/machinelearning/pull/1847#discussion_r241533831) [](ancestors = 241533831)